jtransc-rt-core/src/com/jtransc/JTranscSystem.java

```diff
@@ -104,7 +104,7 @@ static public void debugger() {
 	@HaxeMethodBody("if (!p0) N.debugger();")
 	@JTranscMethodBodyList({
 		@JTranscMethodBody(target = "js", value = "if (!p0) debugger;"),
-		@JTranscMethodBody(target = "cs", value = "if (!pc) System.Diagnostics.Debugger.Break();"),
+		@JTranscMethodBody(target = "cs", value = "if (!p0) System.Diagnostics.Debugger.Break();"),
```

Comment:
Write a test with treeshaking enabled is quite hard, 😂but I managed to do that. Will push after some check.